### PR TITLE
Added MetriCam2 .NET standard project and corresponding Util binary

### PR DESCRIPTION
**Pull request by sisiplac**

### CODE REVIEW

**Comment by f00f** [comment]
@jangernert Wie kann ich helfen?

**Comment by jangernert** [comment]
I don't think I'm the most qualified person to review these changes, since I haven't worked with anything .NET standard related.
Nevertheless I noticed some things that seem odd to me. I'll add them as remarks, but they are more or less questions.

File: MetriCam2/MetriCam2.NetStandard.csproj [hunk 1]
```diff
@@ -23,12 +32,11 @@
 
   <ItemGroup>
     <Folder Include="Properties\" />
-    <Folder Include="Resources\" />
   </ItemGroup>
 
   <ItemGroup>
     <Reference Include="Metrilus.Util.NetStandard">
-      <HintPath>..\..\Util\bin\ReleaseUnix\netstandard2.0\Metrilus.Util.NetStandard.dll</HintPath>
+      <HintPath>..\libraries\Metrilus.Util.NetStandard.dll</HintPath>
```
Comment (jangernert) [inline MetriCam2/MetriCam2.NetStandard.csproj]: `..\libraries\Metrilus.Util.NetStandard.dll` doesn't exist in the project tree. Should it be added?

File: MetriCam2_SDK.sln [hunk 2]
```diff
@@ -299,6 +301,18 @@ Global
...
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x64.Build.0 = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x86.Build.0 = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x64.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x64.Build.0 = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x86.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x86.Build.0 = Release|Any CPU
```
Comment (jangernert) [inline MetriCam2_SDK.sln]: Any reason why some projects default to x64 and some to Any CPU? Shouldn't x64 be preferred especially for new projects?

**Comment by jangernert** [comment]
Okay, all my questions are resolved.

**Pull request merged**